### ISSUE TITLE
[REQ] Pipeline-scoped Meta

### ISSUE THREAD

**Issue by renaud-vmg** [screwdriver-cd/screwdriver#1565]
We need a meta command that sets and gets values across the pipeline.
Use case: iterative process across multiple, CRON-triggered jobs, where we NEED to preserve the "current" status of a sequence when a new event (new commit) gets in.

Stretch goal: possibly with multiple jobs as we could use and update the value "in parallel". There might be need of a "mutex lock" of sorts.
